docker_setup_scripts/centos.sh

```diff
@@ -25,9 +25,17 @@ readonly TOOLSET_PACKAGE_SUFFIXES_CENTOS8=(
 readonly CENTOS7_GCC_TOOLSETS_TO_INSTALL=( 8 9 10 11 )
 readonly CENTOS8_GCC_TOOLSETS_TO_INSTALL=( 9 10 11 )
 
+readonly CENTOS7_ONLY_REPOS=(
+  https://copr.fedorainfracloud.org/coprs/vbatts/bazel/repo/epel-7/vbatts-bazel-epel-7.repo
+)
+readonly CENTOS8_ONLY_REPOS=(
```

Comment:
We don't use CentOS 8, we use AlmaLinux 8, even though this script is called centos.sh. Please rename this to e.g. RHEL8_ONLY_REPOS (and we should probably rename the entire script to `rhel.sh` eventually).